import-export-cli/integration/apim/apim.go

```diff
@@ -1357,7 +1358,7 @@ func (instance *Client) GetAPIByName(name string) (*APIInfo, error) {
 			return &apiResponse.List[0], nil
 		}
 
-		base.WaitForIndexing()
+		time.Sleep(time.Duration(3000) * time.Millisecond)
```

Comment:
Why we have removed `base.WaitForIndexing`? Same logic in L1361 is implemented there

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/957/commits/04388ba15cb22cf8c25006b87d29804211fd5133